### ISSUE TITLE
Duplicate Id on the icons

### ISSUE THREAD

**Issue by tomepajk** [brainstormforce/astra#3356]
In astra/inc/core/class-astra-icons.php, the SVGs Generated have an id of "Layer_1", which causes an A accessibility violation of  1.3.1 Info and Relationships - Element ID is not unique. 

Removing the Id from the SVGs would remove the accessibility violation. 

It can be resolved using the following filter for now.

`add_filter('astra_svg_icon_element', 'make_icons_accessible');

function make_icons_accessible( $output ) {
	$output = str_replace('id="Layer_1"', "");
	return $output;
}`

**Comment by Nikschavan** [comment]
This has been fixed in https://github.com/brainstormforce/astra/pull/3196/

**Comment by thebengalboy** [comment]
Closing the issue as this has been fixed already.
